scripts/checkstack.py

```diff
@@ -180,14 +179,14 @@ def main():
             elif insn in ('rcall', 'call'):
                 cur.noteCall(insnaddr, calladdr, stackusage + 2)
             else:
-                print("unknown call", ref)
+                print(("unknown call", ref))
```

Comment:
passing in a single tuple to `print()` will force the result to be printed as:
```
('unknown call',  <ref>)
```

Which is not desirable, leaving the print as before will give the desired result of:
```
unknown call <ref>
```